plugin/src/compilation/decompiler.ml

```diff
@@ -52,7 +60,17 @@ let get_pushed_names env =
          | Anonymous ->
             failwith "Unexpected Anonymous in get_pushed_names."
          | Name n -> n) names)
-    
+
+(* Functions used with the rewrite tactic. *)
+let is_rewrite_r trm : bool =
+  let eq_term = equal trm in
+  eq_term eq_ind_r || eq_term eq_rec_r (* || eq_term eq_rect_r *)
```

Comment:
Why not ec_rect_r? Why is this commented out?

Reply:
I was going to ask about this. ec_rect_r/ec_rect aren’t available in coq-plugin-lib, so I was wondering if it was worth copying the boilerplate into decompiler.ml. Maybe I can have write access to coq-plugin-lib that way I don’t reimplement anything and keep code in the proper modules?

Reply:
Gave you access